jwst/cube_build/tests/test_wcs.py

```diff
@@ -173,15 +173,15 @@ def test_setup_wcs():
     footprint = (ra_min, ra_max, dec_min, dec_max, lambda_min, lambda_max)
 
     pars_cube = {
```

Comment:
Hey @jemorrison just to capture the reason for the change, why is this change necessary? Does the code behave differently if we use `None` or `0.0`?

Reply:
Is this because of the `assert` for `math.isclose`?

Reply:
As I recall that was the reason -- to make assert for math.isclose work